cmd/aries-agentd/startcmd/start.go

```diff
@@ -50,11 +50,9 @@ const (
 	// AgentWebhookFlagName is the flag name for the webhook command line argument.
 	AgentWebhookFlagName = "webhook-url"
 
-	// AgentWebhookFlagShorthand is the flag shorthand name for the webhook command line argument.
-	AgentWebhookFlagShorthand = "w"
+	agentWebhookFlagShorthand = "w"
 
-	// AgentWebhookFlagUsage is the flag usage text for the webhook command line argument.
-	AgentWebhookFlagUsage = "URL to send notifications to." +
+	agentWebhookFlagUsage = "URL to send notifications to." +
```

Comment:
Just a minor refactor - I realized these were exported when they didn't need to be.